templates/includes/sidebar.html

```diff
@@ -1,4 +1,13 @@
 <ul class="sidebar-nav">
+  {% if user.is_authenticated %}
+    <li><a href="{% url 'taxi:driver-detail' user.id %}">User: {{ user.first_name }} {{ user.last_name }}</a></li>
+    <form id="logout-form" method="POST" action="{% url 'logout' %}">
+      {% csrf_token %}
+      <button type="submit" class="btn btn-link">logout</button>
```

Comment:
The logout form is implemented correctly with a CSRF token for security, which is a good practice.